docs/.vuepress/config.js

```diff
@@ -41,6 +41,46 @@ module.exports = {
         },
       },
     ],
+    ['@simonbrunel/vuepress-plugin-versions', {
+      menu: {
+        items: [
+          {
+            text: 'Documentation',
+            items: [
+              {
+                text: 'Development (master)',
+                link: '/docs/master/',
+              },
+              {
```

Comment:
I would use a [filter](https://github.com/simonbrunel/vuepress-plugin-versions#filters) to replace these two `versions` items:

```js
// filters.js
export default {
  suffix(value) {
    return value ? ` (${value})` : '';
  },
};
```

```js
// config.js
{
  type: 'versions',
  text: '{{version}}{{tag|suffix}}',
  link: '/docs/{{version}}/',
  exclude: '^[013]\\.|2\\.[0-5]\\.',
  group: 'minor',
}
```

The reason is that when releasing `3.1`, you will get 2 entries: `3.1.x (latest)` and `3.0.x (latest)`, etc. By using `{{tag}}`, you display the current tag from npm, so only the most recent release will get it.

Reply:
Thanks, I failed to consider the runtime updates.

Reply:
Looks like there is a subtle difference between the suggested exclude and the implemented one. `^[013]` vs `^[01]`

Reply:
I had initially filtered v3 from the 2nd item, because it was handled separately. Now combined and the exclusion targets versions not having docs (prior to 2.6.0)

Reply:
Right, my `exclude` pattern wasn't correct.